code/game/sound.dm

```diff
@@ -88,6 +88,12 @@
 			var/mob/M = m
 			M.playsound_local(M, null, volume, vary, frequency, falloff, channel, pressure_affected, S)
 
+/proc/direct_sound_to_playing_players(sound_datum) //This is for sending specialized sound datums that playsound() can't account for, i.e. loops
```

Comment:
sound_to_playing_players properly accounts for sound datums, so remove this.

Reply:
Unless this syntax:
`sound_to_playing_players(S = sound('sound/effects/clockcult_gateway_charging.ogg', 1, channel = CHANNEL_JUSTICAR_ARK, volume = 30))`
Is incorrect, it does not in fact properly account for them in testing.

Reply:
it is incorrect, yes.
`sound_to_playing_players(volume = 30, channel = CHANNEL_JUSTICAR_ARK, S = sound('sound/effects/clockcult_gateway_charging.ogg', 1))` is what that line is currently.